app/src/main/java/com/zeapo/pwdstore/git/GitConfigActivity.kt

```diff
@@ -75,8 +75,34 @@ class GitConfigActivity : BaseGitActivity() {
                 e(ex) { "Failed to start GitLogActivity" }
             }
         }
-        binding.gitAbortRebase.setOnClickListener { lifecycleScope.launch { launchGitOperation(BREAK_OUT_OF_DETACHED) } }
-        binding.gitResetToRemote.setOnClickListener { lifecycleScope.launch { launchGitOperation(REQUEST_RESET) } }
+        binding.gitAbortRebase.setOnClickListener {
+            lifecycleScope.launch {
+                launchGitOperation(BREAK_OUT_OF_DETACHED).fold(
+                    onSuccess = {
+                        MaterialAlertDialogBuilder(this@GitConfigActivity)
+                            .setTitle(resources.getString(R.string.git_abort_and_push_title))
+                            .setMessage(resources.getString(
+                                R.string.git_break_out_of_detached_success,
+                                GitSettings.branch,
+                                "conflicting-${GitSettings.branch}-...",
```

Comment:
We're losing the exact branch name here but I think we'll be fine.

Reply:
I think so too